README.md

```diff
@@ -125,6 +127,11 @@ This means that all the options needed for impersonation will be automatically s
 
 Note that the above will NOT WORK for `curl` itself because the curl tool overrides the TLS settings. Use the wrapper scripts instead.
 
+### Notes on dependencies 
+
+If you intend to copy the self-compiled artifacts to another system, or use the [Pre-compiled binaries](#pre-compiled-binaries) provided by us, make sure that all the additional dependencies are met on the target system as well. 
+See notes on [Docker build](INSTALL.md#docker-build) and well as the [Building](INSTALL.md#native-build) section, and the [note about the Firefox version](INSTALL.md#a-note-about-the-firefox-version) subsection.
+
```

Comment:
I think this section is a bit too complicated, throwing the user around to various places.
Given that we only have the nss dependency, can we just list it directly in the installation section? And possible add the link to the notes about the Firefox version. I think that would be more user-friendly.

Reply:
I thought about it, but then assumed that this would make it future-proof, in case you needed more dependencies in the future, also for Chrome build. I'll update that, though. Also, are the binaries you provide compiled statically, such that they actually don't need any additional dependencies installed?